src/Nethermind/Nethermind.Synchronization/SnapSync/SnapServer.cs

```diff
@@ -104,20 +104,18 @@ private bool IsRootMissing(in ValueHash256 stateRoot)
                 default:
                     try
                     {
-                        Hash256 storagePath = new Hash256(requestedPath[0]);
+                        Hash256 storagePath = new Hash256(ValueKeccak.Zero);
+                        requestedPath[0].CopyTo(storagePath.Bytes); // One of the hive tests does not have complete 32 byte for storage path for some reason.
```

Comment:
Avoid double copying?

```suggestion
                        Hash256 storagePath = new Hash256(requestedPath[0].Length == Hash256.Size ? requestedPath[0] : requestedPath[0].PadRight(Hash256.Size));
```